src/index/Index.cpp

```diff
@@ -80,6 +81,18 @@ void Index::createFromFile(const string& filename, bool allPermutations) {
   // also perform unique for first permutation
   createPermutation<IndexMetaDataHmap>(&idTriples, Permutation::Pso, true);
   createPermutation<IndexMetaDataHmap>(&idTriples, Permutation::Pos);
+  if (_entityStats) {
+    ExtVec stats = computeEntityStats(idTriples);
```

Comment:
I think this might have some overlap with `computeExpensiveStatistics()` in [this](https://github.com/ad-freiburg/QLever/pull/105/files) PR